packages/shared/src/components/cards/ActionButtons.tsx

```diff
@@ -157,19 +170,15 @@ export default function ActionButtons({
         </SimpleTooltip>
         {insaneMode && lastActions}
       </ConditionalWrapper>
-      {insaneMode ? (
+      {insaneMode && post.type !== PostType.Freeform ? (
```

Comment:
We can probably look at `internalReadTypes` because all of those don't need read button. For example when collections is merged this would need to be updated.

Reply:
Good idea, will update it.

Reply:
Refactored it now.